services/bstats/bstats-servers.service.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+const BaseJsonService = require('../base-json')
+
+const Joi = require('joi')
+const schema = Joi.object({
+  servers: Joi.number().required(),
+}).required()
+
+module.exports = class BStatsServers extends BaseJsonService {
+  static get route() {
+    return {
+      base: 'bstats/servers',
+      pattern: ':pluginid',
+    }
+  }
+
+  static get defaultBadgeData() {
+    return { label: 'servers' }
+  }
+
+  async handle({ pluginid }) {
+    const { servers } = await this.fetch({ pluginid })
+    return this.constructor.render({ servers })
+  }
+
+  async fetch({ pluginid }) {
+    return this._requestJson({
+      schema,
+      url: `https://bstats-api-format.glitch.me/servers/${pluginid}`,
+    })
+  }
+
+  static render({ servers }) {
+    return {
+      label: 'servers',
```

Comment:
You can exclude `label` here (as the value will always be `servers` and it is defined in `defaultBadgeData`) and you can also move `color` to `defaultBadgeData` as well